frontend/assets/css/tailwind.css

```diff
@@ -18,6 +18,8 @@ MARK: Colors
   --link-text: 0, 92, 184;
   --link-text-hover: 0, 59, 119;
   --highlight: 214, 214, 214;
+  --highlight-darker: 186, 186, 186;
```

Comment:
I added darker and lighter highlight colors, where the matching color to the mode is just the same, @catreedle :) We do the same in activist. I used these instead of the Tailwind greys you used, just so we can match the brand a bit better. Great work on all of this!